src/components/LabCard/styles.jsx

```diff
@@ -0,0 +1,173 @@
+import { makeStyles } from '@mui/styles';
+
+export default makeStyles((theme) => ({
+  healthCard: {
+    color: theme.palette.primary.main,
+    width: '100%',
+    justifyContent: 'center',
+  },
+  card: {
+    backgroundColor: theme.palette.common.blueGreenLight,
+    border: '5px solid',
+    borderColor: 'white',
+    borderRadius: '18px',
+    boxShadow: '0px 4px 8px #00000080',
+    width: '100%',
+    minHeight: '480px',
+    padding: '28px 34.5px;',
+  },
+  cardContent: {
+    padding: 0,
+    '&:last-child': {
+      paddingBottom: 0,
+    },
+  },
+  flexCol: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  patientData: {
+    alignItems: 'flex-start',
+    minHeight: '55px',
+  },
+  nameLabel: {
+    letterSpacing: '0',
+    whiteSpace: 'nowrap',
+  },
+  name: {
+    fontWeight: '700',
+    wordWrap: 'break-word',
+  },
+  dateOfBirthRow: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginLeft: '5.5px',
+    marginTop: '5px',
+  },
+  dateOfBirthLabel: {
+    letterSpacing: '0',
+    whiteSpace: 'nowrap',
+  },
+  dateOfBirth: {
+    fontWeight: '700',
+    whiteSpace: 'nowrap',
+  },
+  eyeOutline: {
+    backgroundSize: '100% 100%',
+    color: theme.palette.common.blueGreenDark,
+    height: '15px',
+    marginBottom: '9px',
+    marginLeft: '10px',
+    width: '22px',
+  },
+  line: {
+    height: '2px',
+    marginTop: '5px',
+    variant: 'middle',
+  },
+  divider: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    alignSelf: 'flex-end',
+    height: '19px',
+    width: '100%',
+  },
+  vaccinationRecordBox: {
```

Comment:
Maybe we should rename some of these objects since this styling is for lab records rather than vaccination records (ex. `labRecordBox`)

Reply:
I renamed all of the `vaccination...` type styles to `lab...` in https://github.com/dvci/shc-web-verifier/pull/77/commits/08573dca3ce250cd16b5d9356d7a6991245f1442